triton/examples/c/hello-world/hello_gcc.rst

```diff
@@ -0,0 +1,32 @@
+Hello world in C
+~~~~~~~~~~~~~~~~
+
```

Comment:
Does it make sense, to have, essentially the same page twice? Wouldn't a tabbed rst page make more sense here? i.e. a "hello_world examples for different languages/compilers" ?

Reply:
Then we would have to combine the page. I created two different pages so that they would be easier to find by the users, but we might rethink this in the future.